src/libtest/Cargo.toml

```diff
@@ -11,6 +11,7 @@ crate-type = ["dylib", "rlib"]
 
 [dependencies]
 getopts = "0.2.19"
+rustc_serialize = { path = "../libserialize", package = "serialize" }
```

Comment:
I think we will want to avoid this in the final version for build simplification reasons, and it looks like not a ton of json is sent so inventing our own serialization format may be reasonable enough?

Reply:
Replaced with exit code :)